src/vs/workbench/browser/parts/activitybar/activitybarActions.ts

```diff
@@ -172,31 +173,37 @@ export class PlaceHolderViewletActivityAction extends ViewletActivityAction {
 
 		const iconClass = `.monaco-workbench > .activitybar .monaco-action-bar .action-label.${this.class}`; // Generate Placeholder CSS to show the icon in the activity bar
 		DOM.createCSSRule(iconClass, `-webkit-mask: url('${iconUrl || ''}') no-repeat 50% 50%`);
-		this.enabled = false;
 	}
 
 	setActivity(activity: IActivity): void {
 		this.activity = activity;
-		this.enabled = true;
 	}
 }
 
 export class PlaceHolderToggleCompositePinnedAction extends ToggleCompositePinnedAction {
 
 	constructor(id: string, compositeBar: ICompositeBar) {
 		super({ id, name: id, cssClass: void 0 }, compositeBar);
-
-		this.enabled = false;
 	}
 
 	setActivity(activity: IActivity): void {
 		this.label = activity.name;
-		this.enabled = true;
 	}
 }
 
 registerThemingParticipant((theme: ITheme, collector: ICssStyleCollector) => {
 
+	const activeBackgroundColor = theme.getColor(ACTIVITY_BAR_FOREGROUND);
```

Comment:
@sandy081 again, I would call it "foreground". The only reason it is "background" is due to CSS rules, but not due to logical reasons that someone would understand that just reads the code.

Reply:
Makes sense to call foreground colour here as it is only for activity bar icons